src/bsp/pc-linux/src/bsp_start.c

```diff
@@ -51,7 +51,12 @@ void OS_Application_Startup(void);
 **    (none)
 */
 
-int main(int argc, char *argv[])
+/// TODO-MAC: [100%] Linking C executable ../../osal_core_UT
+// duplicate symbol '_main' in:
+//    ../../libosal.a(bsp_start.c.o)
+//    ../../libut_bsp.a(bsp_ut.c.o)
+// ld: 1 duplicate symbol for architecture x86_64
+int __main(int argc, char *argv[])
```

Comment:
This should be an easy separate Pull Request. I guess that it works on Linux by accident. Running unit tests should only be possible with a main function from the unit tests, the second one should be compiled and linked in the unit test binary.